tools/tsp-client/test/examples/specification/diagnostics/tspconfig.yaml

```diff
@@ -25,7 +25,7 @@ options:
     flavor: azure
   "@azure-tools/typespec-ts":
     package-dir: "contosowidgetmanager-rest"
-    packageDetails:
+    package-details:
```

Comment:
@catalinaperalta JS emitter would switch to kebab-case style options so camel-case ones would be deprecated soon. So we updated this directly.